erpnext/patches/v15_0/patch_missing_buying_price_list_in_material_request.py

```diff
@@ -9,5 +9,10 @@ def execute():
 		docs = frappe.get_all(
 			"Material Request", filters={"buying_price_list": ["is", "not set"], "docstatus": 1}, pluck="name"
 		)
-		for doc in docs:
-			frappe.db.set_value("Material Request", doc, "buying_price_list", default_buying_price_list)
+		old_limit = frappe.db.MAX_WRITES_PER_TRANSACTION
+		frappe.db.MAX_WRITES_PER_TRANSACTION *= 4
+		try:
+			for doc in docs:
+				frappe.db.set_value("Material Request", doc, "buying_price_list", default_buying_price_list)
+		finally:
+			frappe.db.MAX_WRITES_PER_TRANSACTION = old_limit
```

Comment:
Why are we using query in loop ?

```suggestion
		    frappe.db.set_value("Material Request", {"name": ["in", docs]}, "buying_price_list", default_buying_price_list)

```

Reply:
I will fix the looped query soon, this is kind of an emergency release 😕